src/ui/save-html.js

```diff
@@ -112,6 +113,21 @@ function cleanup(cloneDoc) {
   pub("beforesave", documentElement);
 }
 
+function cleanupComments(node) {
+  // collect first, or walker will cease too early
+  const comments = [...walkTree(node, NodeFilter.SHOW_COMMENT)];
+  for (const comment of comments) {
+    comment.parentNode.removeChild(comment);
```

Comment:
only the hyperHTML comments, no? Sometimes Editors add random comments that they don't mind having on TR.